themes/delphi/assets/css/layouts.scss

```diff
@@ -0,0 +1,36 @@
+.content-grid {

```

Comment:
where is this file imported?

Reply:
Sorry, this was in a half-done status.  layouts is now added to main, but the styles aren't used anywhere.  This this  [code pen](https://codepen.io/chrissco/pen/NWrmVqd) for a usage demo.